ebodac/src/test/java/org/motechproject/ebodac/uitest/test/DisplayOfSubjectVisitInTabularFormatUiTest.java

```diff
@@ -0,0 +1,49 @@
+package org.motechproject.ebodac.uitest.test;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.motech.page.LoginPage;
+import org.motech.test.TestBase;
+import org.motechproject.ebodac.uitest.page.EBODACPage;
+import org.motechproject.ebodac.uitest.page.HomePage;
+import org.motechproject.ebodac.uitest.page.ParticipantEditPage;
+import org.motechproject.ebodac.uitest.page.ParticipantPage;
+
+import static junit.framework.Assert.assertTrue;
+
+
+public class DisplayOfSubjectVisitInTabularFormatUiTest extends TestBase {
+    private LoginPage loginPage;
+    private HomePage homePage;
+    private EBODACPage ebodacPage;
+    private ParticipantPage participantPage;
+    private ParticipantEditPage participantEditPage;
+    private String user;
+    private String password;
+
+    @Before
+    public void setUp() {
+        user = properties.getUserName();
+        password = properties.getPassword();
+        loginPage = new LoginPage(driver);
+        homePage = new HomePage(driver);
+        participantPage = new ParticipantPage(driver);
+        participantEditPage = new ParticipantEditPage(driver);
+        if (homePage.expectedUrlPath() != currentPage().urlPath()) {
```

Comment:
you should not compare two Strings with "!=" or "==", use equals methods instead or if both strings can be null use org.apache.commons.lang.StringUtils.equals